Automattic/woocommerce-payments
# Add support for Japan

Fixes #6479

#### Changes proposed in this Pull Request
Add support for Japan
PR `3565` is required on the server side to test the fees.

#### Testing instructions
**Testing Base fees**

- Go to `WooCommerce` -> `Settings` -> `General`, and set the store Country to **Japan** and currency to **JPY**
- Place an order and verify the base fees are applied.
- Please pick a Japanese card from [Stripe's testing cards](https://stripe.com/docs/testing?testing-method=card-numbers#international-cards)

<img width="417" alt="Screenshot 2023-06-29 at 19 27 23" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/bbff35e7-a7d2-406f-815b-677996cea1f7">

**Testing International and FX fees**
- Go to `WooCommerce` -> `Settings` -> `General`, and set the store currency to USD
- Place an order and verify the fees are applied.
- 
<img width="454" alt="Screenshot 2023-06-29 at 19 28 40" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/dbd4f440-40c5-4423-87af-5e96b89e6f0e">


*

-------------------


**Post merge**
